projects/optic/src/commands/capture/sources/body.ts

```diff
@@ -1,107 +1,48 @@
 import { Readable } from 'stream';
-import { types } from 'util';
 
 export interface CapturedBody {
   contentType: string | null;
   size: number; // size in bytes, defaults to -1
-  stream: Readable | null;
+  body: string | null;
 }
 
 export class CapturedBody {
   static from(
-    source: Readable | Buffer | ArrayBuffer | string | null = null,
+    stringValue: string | null,
     contentType: string | null = null,
     size = 0
```

Comment:
I'm not sure we actually need size anywhere since we're no longer streaming the value